src/app/collection/settings/tab/attributes/attributes-pipe.ts

```diff
@@ -17,18 +17,18 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import {Constraints} from '../constraints';
+import {Pipe, PipeTransform, Injectable} from '@angular/core';
+import {AttributeModel} from '../../../../core/store/collections/collection.model';
 
-export class ConstraintSuggestion {
-
-  public type: string;
-  public color: string;
-  public name: string;
-
-  constructor(constraint: Constraints, listSuggestion: string) {
-    this.type = constraint.type;
-    this.color = constraint.color;
-    this.name = listSuggestion;
+@Pipe({
+  name: 'attributeFilter'
+})
+@Injectable()
+export class AttributesPipe implements PipeTransform {
```

Comment:
I have added many `Attribute*Pipe` classes in my latest pull-request so this name can be confusing. If the pipe is called `attributeFilter`, I would expect class name `AttributeFilterPipe`.

Reply:
Ok